src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -224,8 +225,8 @@ public class MyClass {
 
     <section name="Tool to print Javadoc tree structure">
       <p>
-      Checkstyle can print Abstract Syntax Tree for Java and Javadoc trees. You need to run
-      checkstyle jar file with <b>-J</b> argument, providing java file.
+      Checkstyle can print Abstract Syntax Trees for Java and Javadoc trees. You need to run the
+      checkstyle jar file with the <b>-J</b> argument, providing java file.
```

Comment:
Please remove "the" here, it looks weird in site report

Reply:
But as I stated earlier, this would be grammatically incorrect.

Reply:
It is fine as long as the final rendering doesn't look clumsy